.github/workflows/test.yml

```diff
@@ -235,24 +235,24 @@ jobs:
         env:
           cache-name: cache-gpuocelot-build
         with:
-          path: ${{ github.workspace }}/gpuocelot/ocelot/
+          path: ${{ github.workspace }}/gpuocelot/ocelot
           key: ubuntu22.04-gpuocelot-19626fc00b6ee321638c3111074269c69050e091
-          restore-keys: |
-            ubuntu22.04-gpuocelot-19626fc00b6ee321638c3111074269c69050e091
       - name: Clone/compile gpuocelot
         if: matrix.backend == 'cuda' && steps.cache-build.outputs.cache-hit != 'true'
         run: |
           git clone --recurse-submodules https://github.com/gpuocelot/gpuocelot.git ${{ github.workspace }}/gpuocelot
           cd ${{ github.workspace }}/gpuocelot/ocelot
           git checkout 19626fc00b6ee321638c3111074269c69050e091
-          mkdir build
-          cd build
+          mkdir build && cd build
           cmake .. -Wno-dev -G Ninja -DOCELOT_BUILD_TOOLS=OFF
           ninja
       - name: Install gpuocelot
         if: matrix.backend == 'cuda'
         run: |
-          cd ${{ github.workspace }}/gpuocelot/ocelot/build
+          cd ${{ github.workspace }}/gpuocelot/ocelot
+          mkdir -p build && cd build
+          cmake .. -Wno-dev -G Ninja -DOCELOT_BUILD_TOOLS=OFF
```

Comment:
Duplicated?

Reply:
Yes - I thought the `cmake` and `ninja` calls were required to avoid re-compilation. Turns out there were not. Updated, slimmed down the MR, ready to go (see above).